lib/payment_processor/braintree/webhook_handler.rb

```diff
@@ -69,26 +71,13 @@ def handle_subscription_cancelled
       end
 
       def handle_subscription_charged
-        if original_action.blank?
-          Rails.logger.info("Failed to handle Braintree::WebhookNotification for successful charge on subscription_id '#{@notification.subscription.id}'")
-          return false
-        end
-
-        customer = Payment::Braintree::Customer.find_by(member_id: original_action.member_id)
         record = Payment::Braintree.write_transaction(notification, original_action.page_id, original_action.member_id, customer, false)
```

Comment:
@Tuuleh I'm trying to understand your intentions here. Now that you removed the `if original_action.blank?` and added the `rescue` clause above, we can expect that  the call to `original_action.page_id` will blow up if `original_action` is blank. I don't think this is a very clear way of handling an error. I believe we should explicitly check for the error and respond to it.